src/cli.js

```diff
@@ -156,6 +163,16 @@ let cli = {
               } else {
                 console.log(chalk.green(`Fixes applied, ${ruleName} is now passing`));
               }
+            } else if (answers.disableFiles) {
+              const ruleReport = nibbler.getRuleResults(report, answers.rule);
+              const files = ruleReport.results
+                .filter(result => result.errorCount > 0)
```

Comment:
You can add  _|| result.warningCount > 0_ in filter condition (the warningCount are considered in the other if condition)